src/util-inl.h

```diff
@@ -213,6 +213,20 @@ inline v8::Local<v8::String> OneByteString(v8::Isolate* isolate,
       .ToLocalChecked();
 }
 
+v8::MaybeLocal<v8::String> StringViewToV8String(
```

Comment:
Could this be an overload of `ToV8Value`, similar to the existing `std::string_view` overload? Not a big deal either way but it’s consistent and we do make use of that consistency in other places (e.g. for STL containers over types for which `ToV8Value` is implemented)

Reply:
Ah, I missed the `std::string_view` overload, I was only looking for `*ToString` helpers.

Thanks and done! I also included the length check from the `std::string_view` overload.